medicines/doc-index-updater/.env.example

```diff
@@ -8,4 +8,7 @@ export CREATE_QUEUE_POLICY_KEY=
 export PORT=
 export REDIS_SERVER=
 export REDIS_PORT=
-export REDIS_KEY
\ No newline at end of file
+export REDIS_KEY=
+export SFTP_SERVER=
```

Comment:
You may want to rename these `SENTINEL_SFTP_` etc, because down the line we will use a different source to retrieve PAR files.

Reply:
done something for this now